### ISSUE TITLE
Nashorn deprecation(JDK 11) / removal (JDK 15)

### ISSUE THREAD

**Issue by dstutz** [AdamBien/enhydrator#36]
"The Nashorn engine has been deprecated in JDK 11 as part of JEP 335 and and has been removed from JDK15 as part of JEP 372."

Are there any plans to update enhydrator to utilize GraalVM? https://www.graalvm.org/reference-manual/js/NashornMigrationGuide/

**Comment by ZeTo12** [comment]
Yes, I would also be interested in whether it is possible to use EnHydrator with a current JDK.
What has to be changed? Is there an alternative to GraalVM?
